### DISCUSSION TITLE
Welcome to pytest-docker-tools Discussions!

### DISCUSSION THREAD

**Discussion by Jc2k** [Jc2k/pytest-docker-tools#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mattdavis17** [comment]
Hi there!  Loving the plugin, but had a question that I'm hoping is trivial.  I am looking to override the entrypoint during the fixture's instantiation in a test.  Is this possible? Haven't been able to find anything on it, so I assume it's probably not encourage.

**Reply by Jc2k** [reply]
Because we are a wrapper around [this](https://docker-py.readthedocs.io/en/stable/containers.html), you should just be able to use the `entrypoint` kwarg when defining a container fixture.

**Reply by mattdavis17** [reply]
Thanks for your quick response!  So, I dug into the code a little bit and saw that I could actually pass in the `command` call during the instantiation of the container.  I'll look into the doc's you provided though, thanks for taking the time to reply!
